target/linux/qualcommax/image/ipq60xx.mk

```diff
@@ -168,32 +173,18 @@ endef
 TARGET_DEVICES += tplink_eap610-outdoor
 
 define Device/tplink_eap623-outdoor-hd-v1
-	$(call Device/FitImage)
-	$(call Device/UbiFit)
-	DEVICE_VENDOR := TP-Link
+	$(call Device/tplink_eap6xx-common)
 	DEVICE_MODEL := EAP623-Outdoor HD
 	DEVICE_VARIANT := v1
-	BLOCKSIZE := 128k
-	PAGESIZE := 2048
-	SOC := ipq6018
-	DEVICE_PACKAGES := ipq-wifi-tplink_eap623-outdoor-hd-v1 kmod-phy-realtek
-	IMAGES += web-ui-factory.bin
-	IMAGE/web-ui-factory.bin := append-ubi | tplink-image-2022
+	DEVICE_PACKAGES += ipq-wifi-tplink_eap623-outdoor-hd-v1
 	TPLINK_SUPPORT_STRING := SupportList:\r\nEAP623-Outdoor HD(TP-Link|UN|AX1800-D):1.0\r\n
 endef
 TARGET_DEVICES += tplink_eap623-outdoor-hd-v1
 
 define Device/tplink_eap625-outdoor-hd-v1
-	$(call Device/FitImage)
-	$(call Device/UbiFit)
-	DEVICE_VENDOR := TP-Link
+	$(call Device/tplink_eap6xx-common)
 	DEVICE_MODEL := EAP625-Outdoor HD v1 and v1.6
```

Comment:
version should be moved to DEVICE_VARIANT.

No need to mention v1.6 here. they are compatible. I think there's no hardware change between v1.6 and v1.0(need confirmation).